pulsar-client-api/src/main/java/org/apache/pulsar/client/api/EncryptionKeyInfo.java

```diff
@@ -39,16 +39,16 @@ public EncryptionKeyInfo() {
     }
 
     public EncryptionKeyInfo(byte[] key, Map<String, String> metadata) {
-        this.key = key;
+        this.key = key.clone();
```

Comment:
IMO it is better to add a suppresswarnings here and do not clone the array.
Even if we are not in an hot path

Reply:
OK, I'll ignore the setter/getter of the array.